### PR TITLE
New domain added

### PR DESCRIPTION
**Pull request by soter80**
Slmaxed.com added

### CODE REVIEW

File: script.module.resolveurl/lib/resolveurl/plugins/streamlare.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 class StreamLareResolver(ResolveUrl):
     name = "StreamLare"
-    domains = ["streamlare.com"]
-    pattern = r'(?://|\.)(streamlare\.com)/(?:e|v)/([0-9A-Za-z]+)'
+    domains = ["streamlare.com", "slmaxed.com"] 
+    pattern = r'(?://|\.)(streamlare\slmaxed\.com)/(?:e|v)/([0-9A-Za-z]+)'
```
Comment (Gujal00) [inline script.module.resolveurl/lib/resolveurl/plugins/streamlare.py:28]: Pattern is not right, did you test if it works?
It should be like this for it to work
`pattern = r'(?://|\.)((?:streamlare|slmaxed)\.com)/(?:e|v)/([0-9A-Za-z]+)'`

Comment (soter80) [reply script.module.resolveurl/lib/resolveurl/plugins/streamlare.py:28]: I test it and it not work unfortunstely

Comment (Gujal00) [reply script.module.resolveurl/lib/resolveurl/plugins/streamlare.py:28]: Can you give me a sample embed in slmaxed.com

Comment (Gujal00) [reply script.module.resolveurl/lib/resolveurl/plugins/streamlare.py:28]: Yes can't think of a way to make it work as they have enabled cloudflare browser fingerprinting
`URLError Error - From: StreamLare Link: https://slmaxed.com/e/XXXXXXXX: <urlopen error [WinError 10054] An existing connection was forcibly closed by the remote host>`

**Comment by Gujal00** [comment]
https://check-your-website.server-daten.de/?q=www-bew71760.ssl0d.com

**Pull request closed without merging**